ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/TokenClaimBehaviorType.kt

```diff
@@ -0,0 +1,26 @@
+package net.corda.v5.ledger.utxo
+
+import net.corda.v5.base.annotations.CordaSerializable
+
+/**
+ * Defines the token claim behavior when finalizing a transaction.
+ */
+@CordaSerializable
+enum class TokenClaimBehaviorType {
+    /**
+     * Automatically claim any tokens output during finalization. Any claimed tokens are returned via the
+     * [FinalizationResult.tokenClaim] property. This is the default option.
```

Comment:
Is that the correct default option? I guess this only  makes sense if your states are actually tokens that can be claimed/selected?
Also, if that isn't implemented, having this as default means that the default behaviour will change when this gets implemented.